.gitignore

```diff
@@ -5,3 +5,4 @@ erl_crash.dump
 /doc
 /docs
 /bench/snapshots
+.elixir_ls/
```

Comment:
Or, you can ignore `.elixir_ls` outside of this repo by configuring a global ignore file: http://egorsmirnov.me/2015/05/04/global-gitignore-file.html